src/mixins/portfolio.js

```diff
@@ -405,8 +508,10 @@ export const createPorfolioMixin = ({
     changeTab(tab) {
       if (!tabOptions[tab]) return;
       this.syncRouteForTab(tab);
+      this.nftTypeFilter = NFT_TYPE_FILTER_OPTIONS.ALL;
+      this.nftCreatorFilter = [];
     },
-    handleNFTClassListFilteringChange({ type, value }) {
+    handleNFTClassListCreatorChange({ type, value }) {
       logTrackerEvent(
         this,
         'portfolio',
```

Comment:
Since the type always be `creator` now, we just just omit the `type` and hardcode `creator`